lib/school.rb

```diff
@@ -0,0 +1,46 @@
+class School
+
+  attr_reader :start_time,
+              :total_hours,
+              :name
+
+  def initialize(start_time, total_hours)
+    @start_time = start_time
+    @total_hours = total_hours
+    @names =[]
+  end
+
+  def hours_in_school_day()
```

Comment:
Nitpick: we don't need the parens in method definition here since we are not passing in any parameters

Reply:
This entire method isn't needed since we can rename the `total_hours` to `hours_in_school_day`